### PR TITLE
Fixed Maturity Age 

### LINKED ISSUE #408: Maturity age needs to be fixed.
Change the maturity age in parameters.sol to 95.

### PR DESCRIPTION
**Pull request by GauravJain9**
Fixes #408 
Changed the maturity age (maxAge) from 100 to 95

### CODE REVIEW

File: contracts/Core/Parameters.sol [hunk 1]
```diff
@@ -18,7 +18,7 @@ contract Parameters is ACL, Constants {
     uint16 public epochLength = 300;
     uint16 public exposureDenominator = 1000;
     uint16 public gracePeriod = 8;
-    uint32 public maxAge = 100 * 10000;
+    uint32 public maxAge = 95 * 10000;
```
Comment (abhishekvispute) [inline contracts/Core/Parameters.sol:21]: Hey
IMO, we should keep max age as 100 *10000 only.
And update maturity array with values for 100
https://github.com/razor-network/contracts/blob/c248922e0e72e8c69a7df357e1eac17497afe3ab/contracts/Core/storage/StakeStorage.sol#L12

Updating max age to 95 was a fix we did on live net, as we couldn't update this array there, now we can do

Whats your opinion on this guys ?
@hrishikeshio @SkandaBhat @SamAg19

Comment (SkandaBhat) [reply contracts/Core/Parameters.sol:21]: Sounds good. @GauravJain9 please note

Comment (GauravJain9) [reply contracts/Core/Parameters.sol:21]: yeah @SkandaBhat  shall we keep it 499 or 500 ?  How is this calculated ?

Comment (hrishikeshio) [reply contracts/Core/Parameters.sol:21]: use this`[50, 70, 86, 100, 111, 122, 132, 141, 150, 158, 165, 173, 180, 187, 193, 200, 206, 212, 217, 223, 229, 234, 239, 244, 250, 254, 259, 264, 269, 273, 278, 282, 287, 291, 295, 300, 304, 308, 312, 316, 320, 324, 327, 331, 335, 339, 342, 346, 350, 353, 357, 360, 364, 367, 370, 374, 377, 380, 384, 387, 390, 393, 396, 400, 403, 406, 409, 412, 415, 418, 421, 424, 427, 430, 433, 435, 438, 441, 444, 447, 450, 452, 455, 458, 460, 463, 466, 469, 471, 474, 476, 479, 482, 484, 487, 489, 492, 494, 497, 500, 502]
`

code
https://colab.research.google.com/drive/1ORFup2QO9IOIZIr2IZOCjhaZHUbWeWgE?usp=sharing

**Review by abhishekvispute — APPROVED** [review]

**Review by SkandaBhat — APPROVED** [review]
LGTM

**Pull request merged**